amd64/root/firmware-update.sh

```diff
@@ -14,7 +17,7 @@ echo " "
 echo "Listing attached devices..."
 echo " "
 
-/usr/bin/GCFFlasher_internal -l
+$FLASHER -l
 
```

Comment:
I would love to see a prefiled variable with the device path if there is just 1 result. This should be in most cases the result and speed up the process.